integration-tests/src/mock_network/README.md

```diff
@@ -0,0 +1,15 @@
+# mock-network
+This crate hosts libraries to start a test env for a single node by replacing the network module with a mock network environment. 
+The mock network environment simulates the interaction that the client will usually have with other nodes by 
+responding to the client's network messages and broadcasts new blocks. The mock network reads a pre-generated chain 
+history from storage.
+
+The crate has two files and another binary sits in ../bin/start_mock_network
+
+## mod.rs
+Implements `ChainHistoryAccess` and `MockPeerManagerActor`, which is the main 
+components of the mock network.
+## setup.rs
+Provides functions for setting up a mock network from configs and home dirs.
+## start_mock_network.rs
+A binary that starts a mock network from the home dir of an existing node
```

Comment:
maybe also add the example command line here?

Also - the part that confused me is 'sync' vs 'no sync' (let me know if I understand it correctly:
if no sync:
- everyone starts from 'genesis_height'
- the 'mock' network will keep 'producting' block until 'target_height'.

If sync:
- the 'mock' network starts from 'target_height' and does NOT produce any more blocks, right?
- and the 'node' is just trying to sync 


(then the option I'm missing -- but can be added in the next PR is the mixture, where:

- the 'mock' network starts from height X, and keeps producing 1 block per second until height Y
- the 'node' tries to 'sync' and then catchup and keep getting the new blocks that are produced.
(this last mode is the most similar to what is happening in production today).

Reply:
Yeah that's right. I do plan to add these options later, not now because we can't support it yet, because the client can only start from genesis height for now.

Later, in the full version, the binary take  `client_start_height`, `target_height`. 
If no sync, everyone starts from 'client_start_height' and the 'mock' network will keep 'producting' block until 'target_height'.
If sync, the binary can take another argument `network_start_height`. Client starts from `client_start_height` and the network starts from `network_start_height` and will keep producing blocks until `target_height`.

Reply:
Example command line will be printed by the binary, because of how clap works.